packages/git/src/browser/history/git-history-contribution.ts

```diff
@@ -43,6 +46,25 @@ export class GitHistoryContribution extends AbstractViewContribution<GitHistoryW
         });
     }
 
+    @postConstruct()
+    init() {
```

Comment:
By convention, we use `protected` for the `postConstruct` methods. It was just a side-note.